generator/required.go

```diff
@@ -1,12 +1,16 @@
 package generator
 
+// ExcludedFields is a list of tags which not will be at generated messages
+// because they already exists in base structure
```

Comment:
ExcludedFields is a list of tags which **will be omitted** at generated messages because they already exists in base structure